Apps/Src/Contactor.c

```diff
@@ -22,25 +19,14 @@ void Contactor_Init(void) {
  * @return  None
  */
 void Contactor_On(Contactors_t contactorChoice) {
-    // set output pins to start outputing with a duty cycle set by PWM_ON_TIME in the header file
-    switch (contactorChoice)
-    {
-    case ARRAY_CONTACTOR:
-        BSP_PWM_Set(ARRAY_CONTACTOR_OUT, PWM_ON_TIME);
-        break;
-    case HVHIGH_CONTACTOR:
-        BSP_PWM_Set(HVHIGH_CONTACTOR_OUT, PWM_ON_TIME);
-        break;
-    case HVLOW_CONTACTOR:
-        BSP_PWM_Set(HVLOW_CONTACTOR_OUT, PWM_ON_TIME);
-        break;
-    case ALL_CONTACTORS: //c1 and cfan are the only two possible contactors, so we operate on both
-        BSP_PWM_Set(CFAN, PWM_ON_TIME);
-        BSP_PWM_Set(C1, PWM_ON_TIME);
-        break;
-    default:
-        break;
-    }
+	// set output pins to start outputing with a duty cycle set by PWM_ON_TIME in the header file
+	if(contactorChoice == ALL_CONTACTORS){
+		//c1 and cfan are the only two possible contactors, so we operate on both
+		BSP_PWM_Set(CFAN, PWM_ON_TIME);
+		BSP_PWM_Set(C1, PWM_ON_TIME);
+	}else{
+		BSP_PWM_Set(contactorChoice, PWM_ON_TIME);
+	}
```

Comment:
You can't just put in contactorChoice because that is a number from 1-4. I think this code would be better if you either had the enum for ``Contactors_t`` or the #defines for ``X_CONTACTOR_OUT`` in order to prevent this confusion.